tutorials/scripting/gdextension/gdextension_file.rst

```diff
@@ -0,0 +1,95 @@
+.. _doc_gdextension_file_sections:
+
+The .gdextension file
+=====================
+
+Introduction
+------------
+
+The ``.gdextension`` file in your project contains the instructions for the editor how to parse
+the GDExtensions in projects. The instructions are separated into specific sections. This page
+should give you a quick overview of the different options available to you. For an introduction
+how to get started with GDExtensions take a look at the :ref:`GDExtension C++ Example <doc_gdextension_cpp_example>`.
+
+Configuration section
+---------------------
+
++-------------------------------+------------------------------------------------------------------------------------------------------+
+| Instruction                   | Description                                                                                          |
++===============================+======================================================================================================+
+| **entry_smbol**               | Name of the entry function of the loaded GDExtension. This function should be defined in             |
+|                               | the ``register_types.cpp`` file. Adding this is necessary for the extension to work.                 |
++-------------------------------+------------------------------------------------------------------------------------------------------+
+| **compatability_minimum**     | Minimum compatible version. This prevents older editor binaries from loading the extension.          |
++-------------------------------+------------------------------------------------------------------------------------------------------+
+
+Libraries section
+-----------------
+
+In this section you can set the paths to your compiled binaries of your GDExtension libraries.
+By setting feature flags you can filter which version should be loaded and exported with your
+game depending on which feature flags are active.
```

Comment:
I think we should add something about how all feature flags on each line must match (so, `macos.debug` means that it will be loaded if Godot has both "macos" and "debug" feature flags), and that each line is evaluated in order from top-to-bottom.